### PR TITLE
{AKS} Fix the test failure of network isolated cluster feature

### PR DESCRIPTION
**Pull request by bingosummer**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

```
az aks create --resource-group={resource_group} --name={name} --bootstrap-artifact-source Cache --bootstrap-container-registry-resource-id {acr_id} --outbound-type=none/block
```

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by FumingZhang** [comment]
> Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=107852381&view=results) to validate the change.
> 
> * test_aks_network_isolated_cluster

Case failed with the following error
```
E                   azure.cli.core.azclierror.BadRequestError: (InvalidNetworkIsolatedClusterConfiguration) When outbound type is none or block, the cluster must be private cluster or enables apiserver vnet integration
E                   Code: InvalidNetworkIsolatedClusterConfiguration
E                   Message: When outbound type is none or block, the cluster must be private cluster or enables apiserver vnet integration
```

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -9,6 +9,10 @@ If there is no rush to release a new version, please just add a description of t
 
 To release a new version, please select a new version number (usually plus 1 to last patch version, X.Y.Z -> Major.Minor.Patch, more details in `\doc <https://semver.org/>`_), and then add a new section named as the new version number in this file, the content should include the new modifications and everything from the *Pending* section. Finally, update the `VERSION` variable in `setup.py` with this new version number.
 
+Pending
++++++++
+* Fix the test failure of network isolated cluster feature.
+
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:15]: May omit the note here as you only modified test cases, which is not visible to external users.

**Comment by FumingZhang** [comment]
> > Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=107852381&view=results) to validate the change.
> > 
> > * test_aks_network_isolated_cluster
> 
> Case failed with the following error
> 
> ```
> E                   azure.cli.core.azclierror.BadRequestError: (InvalidNetworkIsolatedClusterConfiguration) When outbound type is none or block, the cluster must be private cluster or enables apiserver vnet integration
> E                   Code: InvalidNetworkIsolatedClusterConfiguration
> E                   Message: When outbound type is none or block, the cluster must be private cluster or enables apiserver vnet integration
> ```

Requeued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=107949899&view=results)

File: src/aks-preview/HISTORY.rst (hunk 1 above)
Comment (bingosummer) [reply src/aks-preview/HISTORY.rst]: thanks, removed

**Comment by bingosummer** [comment]
Requeued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=107951489&view=results)

**Review by FumingZhang — APPROVED** [review]
LGTM

**Review by FumingZhang — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**